CREATING_COMPOUND_COMPONENTS.md

```diff
@@ -517,7 +521,7 @@ The full code can be found here: https://codesandbox.io/s/configurable-disclosur
 
 ## Model Composition
 
-Our example isn't fully accessible yet. The Disclosure target needs a `aria-controls` attribute to tie the target and content in the accessibility tree. This is done by the use of IDREFs (string IDs that start with a letter). We could add an  `id` to our model, but it is extremely common so let's make a new model and compose from it instead. We'll later use this model in a reusable behavioral hook.
+Our example isn't fully accessible yet. The Disclosure target needs a `aria-controls` attribute to tie the target and content in the accessibility tree. This is done by the use of IDREFs (string IDs that starts with a letter). We could add an  `id` to our model, but it is extremely common so let's make a new model and compose from it instead. We'll later use this model in a reusable behavioral hook.
```

Comment:
I think `start` may have been the appropriate tense?

Reply:
I think it is "string ID that starts" or "string IDs that start"